curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/6538935e2ab721beedb137c4.md

```diff
@@ -1,19 +1,19 @@
 ---
 id: 6538935e2ab721beedb137c4
-title: Step 25
+title: Step 23
 challengeType: 0
-dashedName: step-25
+dashedName: step-23
 ---
 
 # --description--
 
-Finally, join the results from the array reversal. Use a `"-"` as the separator. Remember to use method chaining.
+Like in the previous step, use method chaining to `split`, `reverse`, and `join` the array.
 
 Test out your changes by selecting the `Year, Month, Day` option from the dropdown menu. The date should now be displayed in the format `yyyy-mm-dd`.
 
 # --hints--
 
-You should use the `.join()` method to join the reversed array elements into a string and use a `"-"` as a separator.
+The `.join()` and `.split()` methods should use a `"-"` as a separator.  Make sure to use `.reverse()` at the end.
 
 ```js
 assert(code.match(/\.join\(\s*('|")-\1\s*\)/g));
```

Comment:
This test should be updated to check that all three things are used. Or more tests should be added to check them individually.